logbook-api/src/main/java/org/zalando/logbook/ForwardingStrategy.java

```diff
@@ -0,0 +1,42 @@
+package org.zalando.logbook;
+
+import org.apiguardian.api.API;
+import org.zalando.logbook.attributes.RequestAttributesExtractor;
+
+import java.io.IOException;
+
+import static org.apiguardian.api.API.Status.STABLE;
+
+@API(status = STABLE)
+public interface ForwardingStrategy extends Strategy {
```

Comment:
What's the idea for this strategy? Is it related to the JWT?

Reply:
I'll use it later on when I implement it in Spring Boot module.
Also, our clients can use like this:

https://github.com/msdousti/LogbookJwtTester/blob/45d2b385fb937e44f13ed660fcc8648fd2faa48b/src/main/java/org/example/Main.java#L66-L77

Reply:
Removed `ForwardingStrategy`.